### ISSUE TITLE
FSR - Unauthenticated debt selection designs

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#18701]
## Issue Description
For the unauthenticated FSR, we will need to make some adjustments to several pages that would normally be pre-filled for authenticated users. 

---
## Tasks
- [x] Alternative designs created for C1P2: Debt selection
- [x] Sync designs to UXPin
- [x] Create frontend ticket 

## Acceptance Criteria
- [x] Designs synced to UXPin and linked in this ticket
- [x] Frontend ticket created and linked in this ticket

**Comment by rileyorr** [comment]
Added [the unauthenticated debt selection page](https://preview.uxpin.com/b4745b18da61eee6b2c74af081afe3aec6cbd685#/pages/136312482/simulate/no-panels?mode=i) to the UXPin. I went through a few iterations of how we could gather debt data, but at the end of the day the DMC should be able to find the Veterans' debt info so long as they provide their SSN/file number and select which debt type(s) the request is for. I'll create a frontend ticket first thing tomorrow morning!
